src/lib/index.ts

```diff
@@ -50,6 +54,7 @@ const styles = {
 export const viz = {
   ...sources,
   ...basemaps,
+  ...widgets,
```

Comment:
As I said in the previous, and I think this namespace won't have much discussion --> let's go for an `export const widget`?